.env.local

```diff
@@ -2,6 +2,7 @@ NEXT_PUBLIC_APPINSIGHTS_INSTRUMENTATIONKEY=2f1d6a9c-1b88-4f3d-bba1-64b11ffd2362
 NEXT_PUBLIC_GET_SUBMISSIONS_URL=https://dddperth-functions-test.azurewebsites.net/api/GetSubmissions
 NEXT_PUBLIC_SUBMIT_VOTE_URL=https://dddperth-functions-test.azurewebsites.net/api/SubmitVote
 NEXT_PUBLIC_GET_AGENDA_URL=https://dddperth-functions-test.azurewebsites.net/api/GetAgenda
+NEXT_PUBLIC_GET_AGENDA_SCHEDULE_URL=https://dddperth-functions-test.azurewebsites.net/api/GetAgendaSchedule
```

Comment:
Need someone to add this to the pipeline ... I don't have permission. 
@mzaatar